### DISCUSSION TITLE
v2.0.0 Technical Preview 6

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2114] [分享 / Share]
- 搜索栏可以使用 <kbd>/</kbd> 全局唤起了. (安装了`快捷键扩展`时)
- 自定义顶栏功能同步至 `v1.12.13`.
- 新增组件 `v1 设置迁移`, 可以导入 v1 设置了, README 中有使用说明.
- 支持[合集包](https://github.com/the1812/Bilibili-Evolved/blob/v2/doc/features/pack/pack.md)安装, 设置面板中的批量安装可以接受任意类型的功能.
- `关于`面板中有了更多的链接, 并显示更详细的版本信息.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.0-tp6'>v2.0.0 Technical Preview 6</a>.</em>

**Comment by SekiBetu** [comment]
这次的更新日志怎么没有像 Preview 5一样说明组件迁移和进度等内容了

**Reply by the1812** [reply]
5 已经基本迁完了, 这次只是功能更新
剩下几个之后我再发到这里

**Comment by kethoven** [comment]
好用了不少！

**Comment by Geeyun-JY3** [comment]
导入 v1 设置报错：
`Unexpected token F in JSON at position 0`
![image](https://user-images.githubusercontent.com/49396421/127812043-f01e7d08-92ff-4a79-949c-24d17c7cd8c9.png)
设置项：
[bilibili-evolved-settings.zip](https://github.com/the1812/Bilibili-Evolved/files/6914681/bilibili-evolved-settings.zip)

另外建议内置 `v1 设置迁移` 组件

**Reply by the1812** [reply]
看下 console?

**Reply by Geeyun-JY3** [reply]
```
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
VM868:1 下载功能列表中
VM868:1 Failed to fetch the1812/Bilibili-Evolved@2 from GitHub.
userscript.html?name=Bilibili%20Evolved%20(v2).user.js&id=ab014c3e-3595-4f48-917d-44e5a7f7f433:226 SyntaxError: Unexpected token F in JSON at position 0
    at JSON.parse (<anonymous>)
    at eval (eval at <anonymous> (chrome-extension://gcalenpjmijncebpfijmoaglllgpjagf/userscript.html?name=Bilibili%2520Evolved%2520(/v2).user.js&id=ab014c3e-3595-4f48-917d-44e5a7f7f433), <anonymous>:1:2516)
    at async Object.run (eval at <anonymous> (chrome-extension://gcalenpjmijncebpfijmoaglllgpjagf/userscript.html?name=Bilibili%2520Evolved%2520(/v2).user.js&id=ab014c3e-3595-4f48-917d-44e5a7f7f433), <anonymous>:1:2256)
    at async a.runAction (chrome-extension://gcalenpjmijncebpfijmoaglllgpjagf/userscript.html?name=Bilibili%2520Evolved%2520(/v2).user.js&id=ab014c3e-3595-4f48-917d-44e5a7f7f433:259:20739)
a @ userscript.html?name=Bilibili%20Evolved%20(v2).user.js&id=ab014c3e-3595-4f48-917d-44e5a7f7f433:226
DevTools failed to load source map: Could not load content for https://s1.hdslb.com/bfs/static/jinkela/long/js/sentry/bundle.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://s1.hdslb.com/bfs/static/jinkela/international-home/international-home.c06a8fc1db2de82eb27f56ea1acb93179753895d.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://s1.hdslb.com/bfs/static/jinkela/long/js/sentry/vue.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://s1.hdslb.com/bfs/static/jinkela/international-home/css/international-home.0.c06a8fc1db2de82eb27f56ea1acb93179753895d.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://s1.hdslb.com/bfs/static/jinkela/international-home/css/international-home.1.c06a8fc1db2de82eb27f56ea1acb93179753895d.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://s1.hdslb.com/bfs/static/jinkela/international-home/1.international-home.c06a8fc1db2de82eb27f56ea1acb93179753895d.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```
![image](https://user-images.githubusercontent.com/49396421/127813315-89e9e57f-4e96-4200-9153-8df7d99ef8d6.png)

**Reply by the1812** [reply]
你的更新源是不是还没换成 GitHub

**Reply by Geeyun-JY3** [reply]
> 你的更新源是不是还没换成 GitHub

抱歉看漏了。换成 GitHub 就没问题了。

**Comment by Geeyun-JY3** [comment]
<kbd>Shift</kbd> + <kbd>P</kbd> 唤起一个 Bilibili Evolved 的搜索栏算是间接实现了 #1197 吗？HTML5Player 预设的画中画键位就是 <kbd>Shift</kbd> + <kbd>P</kbd>，会有冲突问题。
而且为啥不用 <kbd>/</kbd> 捏？感觉这更通用啊。

**Reply by the1812** [reply]
当时随手写的 <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> (同 VS Code), 结果和 Chrome DevTools 撞车了就先把 <kbd>Ctrl</kbd> 去了
<kbd>/</kbd>应该比较合适, 我之后再改改.

**Comment by jiacchen** [comment]
[bilibili-evolved-settings.json.zip](https://github.com/the1812/Bilibili-Evolved/files/6921412/bilibili-evolved-settings.json.zip)
我从v1迁移之后，发现单击播放器会自动变成双击（播放之后瞬间暂停），好像是`双击全屏`的问题。

（还有请问Safari是还没适配吗，似乎启用之后左侧中间也不会显示按钮）

**Reply by RingingVolcano** [reply]
> [bilibili-evolved-settings.json.zip](https://github.com/the1812/Bilibili-Evolved/files/6921412/bilibili-evolved-settings.json.zip)
> 我从v1迁移之后，发现单击播放器会自动变成双击（播放之后瞬间暂停），好像是`双击全屏`的问题。
> 
> （还有请问Safari是还没适配吗，似乎启用之后左侧中间也不会显示按钮）

把这个打开就行了
![image](https://user-images.githubusercontent.com/72140353/128424192-3e4d1b21-3569-4fef-b5f0-9f8ae44d08bc.png)

**Comment by Unknown-shu** [comment]
添加组件错误
版本号：v2.0.0-tp6-49-g44cc8afc
![image](https://user-images.githubusercontent.com/64715642/130324781-a16001c8-cb95-4312-b14c-2c465cf924a6.png)
![image](https://user-images.githubusercontent.com/64715642/130324804-47079465-788c-486e-aeb3-7a34ccc5fc97.png)

**Reply by the1812** [reply]
需要梯子的

**Reply by Unknown-shu** [reply]
> 需要梯子的

那就好办了，阿里嘎多
